### ISSUE TITLE
Trouble including functions in Riemann 0.3.3

### ISSUE THREAD

**Issue by maddenp** [riemann/riemann#954]
Following the instructions here -- http://riemann.io/howto.html#including-functions-from-multiple-files -- I have a problem with Riemann 0.3.3 that I cannot reproduce with Riemann 0.3.2, namely:
```
# riemann-0.3.3/bin/riemann 
INFO [2019-08-23 02:37:06,321] main - riemann.bin - Loading /root/riemann-0.3.3/etc/riemann.config
INFO [2019-08-23 02:37:06,394] main - riemann.bin - PID 893
ERROR [2019-08-23 02:37:06,405] main - riemann.bin - Couldn't start
clojure.lang.Compiler$CompilerException: java.io.FileNotFoundException: Could not locate foo/bar__init.class or foo/bar.clj on classpath., compiling:(/root/riemann-0.3.3/bin/../etc/riemann.config:1:1)
...
```
while
```
# riemann-0.3.2/bin/riemann 
INFO [2019-08-23 02:37:54,176] main - riemann.bin - Loading /root/riemann-0.3.2/etc/riemann.config
INFO [2019-08-23 02:37:54,235] main - riemann.bin - PID 908
INFO [2019-08-23 02:37:54,492] clojure-agent-send-off-pool-1 - riemann.transport.websockets - Websockets server 127.0.0.1 5556 online
INFO [2019-08-23 02:37:54,691] clojure-agent-send-off-pool-0 - riemann.transport.udp - UDP server 127.0.0.1 5555 16384 -1 online
INFO [2019-08-23 02:37:54,700] clojure-agent-send-off-pool-2 - riemann.transport.tcp - TCP server 127.0.0.1 5555 online
INFO [2019-08-23 02:37:54,701] main - riemann.core - Hyperspace core online
```
I've simply downloaded and extracted the tarballs for the two releases, added the line
```
(require '[foo.bar :refer :all])
```
to `etc/riemann.config` in each tree, and added
```
# cat riemann-0.3.3/etc/foo/bar.clj 
(ns foo.bar)
```
(and the same under `riemann-0.3.2`). This is
```
# java -version
openjdk version "10" 2018-03-20
OpenJDK Runtime Environment 18.3 (build 10+44)
OpenJDK 64-Bit Server VM 18.3 (build 10+44, mixed mode)
```
Are there release notes somewhere I might have missed, or have I otherwise made a mistake? Thanks in advance.

**Comment by jamtur01** [comment]
I'm guessing this is probably related to #951. That's still not quite resolved but could you test with that branch?

**Comment by maddenp** [comment]
I can confirm that that branch does not exhibit the bad behavior.

**Comment by jamtur01** [comment]
Released in 0.3.4
